### ISSUE TITLE
为什么使用mbv3_small的预训练会出现乱码

### ISSUE THREAD

**Issue by jideee** [shaoshengsong/MobileNetV3-SSD#30]
我使用mbv3_small.pth 加载到这个程序当中为什么会出现乱码 下面是我加载的命令

python3.6 train_ssd.py --datasets /home/cjd/ssd.pytorch/data/VOCdevkit/VOC2007/ --validation_dataset /home/cjd/ssd.pytorch/data/VOCdevkit/VOC2007/ --net mb2-ssd-lite --base_net models/mbv3_small.pth --batch_size 8 --scheduler cosine --lr 0.01 --t_max 200 --validation_epochs 5 --num_epochs 200

就一直出现乱码 不知道是v3的预训练不行还是 此程序不行

请各位指点下  谢谢
